tests/integration.coffee

```diff
@@ -946,8 +946,12 @@ describe 'SDK Integration Tests', ->
 
 					describe 'resin.models.device.getDeviceUrl()', ->
 
+						it 'should eventually be a url containing full length uuid', ->
+							promise = resin.models.device.getDeviceUrl(@device.uuid.slice(0, 7))
+							m.chai.expect(promise).to.eventually.match(/[a-z0-9]{62}/)
+
 						it 'should eventually be an absolute url', (done) ->
-							resin.models.device.getDeviceUrl(@device.uuid).then (deviceUrl) ->
+							resin.models.device.getDeviceUrl(@device.uuid.slice(0, 7)).then (deviceUrl) ->
```

Comment:
Why is this being changed to use a shorter uuid if the above test is already handling the shorter uuid?